packages/vue/src/virtual-scroll/index.ts

```diff
@@ -1,5 +1,4 @@
-import VirtualScroll from './src/index'
-
+import VirtualScroll from './src/index.js'
```

Comment:
不应该导入 index.js 文件 @necoxrr 
```suggestion
import VirtualScroll from './src/index'
```

Reply:
好的，之后PR会进行更改